index.js

```diff
@@ -74,22 +100,30 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 6", "color: red")
 
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
-// // => "Duane"
-// // => "Cell: 555-123-4567"
-// // => "Office: 555-456-7890"
-// // => "Liza"
-// // => "Cell: 555-234-5678"
-// // => "Office: 555-567-1234"
-// console.log("%c----------", "color: red") 
+function printNameAndPhones(users) {
+  users.forEach(user => {
+    console.log(user.name)
+    console.log(`Cell: ${user.phones["cell"]}`)
+    console.log(`Office: ${user.phones["office"]}`)
+  });
```

Comment:
Love to see the arrow function syntax! We'll go over that a bit later this week, but arrow functions are really nice for callbacks.

For accessing the user's phones, you can also use dot notation (since `.phones` is also an object):

```js
    console.log(`Cell: ${user.phones.cell}`)
    console.log(`Office: ${user.phones.office}`)
```